### PR TITLE
Push implementation of service out of ert

### PR DESCRIPTION
**Pull request by eivindjahren**
In order to avoid install webviz_ert from ert,
the deprecated ert_hooks is removed and the vis service
executable from ert is moved to webviz_ert

This PR has to be coordinated with https://github.com/equinor/ert/pull/3757

## Pre review checklist

- [x] Added appropriate labels

### CODE REVIEW

File: webviz_ert/__main__.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            if page["experimental"]:
+                return experimental_mode
+        return True
+
+    new_config_dict["pages"] = [
+        page
+        for page in new_config_dict["pages"]
+        if filter_experimental_pages(page, experimental_mode)
+    ]
+
```
Comment (hnformentin) [inline webviz_ert/__main__.py:61]: Clean Code -> Vertical order -> function below a function that does the calling. Is here an exception?

**Comment by hnformentin** [comment]
The error seems not to be a flakiness:

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
webviz-config 0.5.0a0 requires bleach[css]>=5; python_version >= "3.7", but you have bleach 4.1.0 which is incompatible.

ModuleNotFoundError: No module named 'webviz_ert.ert_hooks'
ERT crashed unexpectedly with "No module named 'webviz_ert.ert_hooks'".

**Comment by eivindjahren** [comment]
> The error seems not to be a flakiness:
> 
> ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts. webviz-config 0.5.0a0 requires bleach[css]>=5; python_version >= "3.7", but you have bleach 4.1.0 which is incompatible.
> 
> ModuleNotFoundError: No module named 'webviz_ert.ert_hooks' ERT crashed unexpectedly with "No module named 'webviz_ert.ert_hooks'".

I thought I had already fixed that. I will have a closer look.

File: webviz_ert/__main__.py (hunk 1 above)
Comment (eivindjahren) [reply webviz_ert/__main__.py:61]: No, I simply move this function from ert to webviz_ert. I'll clean it up :)

**Comment by eivindjahren** [comment]
Jenkins, test this please!

**Comment by eivindjahren** [comment]
The failure on jenkins seems to be due to ert picking up the hook, even though we have removed it. I think this might be due to the bleeding environment this is being deployed into being out of sync. I suggest ignoring this for now and hopefully it will be fixed with rebuilding bleeding.

File: webviz_ert/__main__.py (hunk 1 above)
Comment (eivindjahren) [reply webviz_ert/__main__.py:100]: Fixed now.

**Review by pinkwah — APPROVED** [review]

**Pull request merged**